Microsoft.Toolkit.Uwp/Helpers/ObjectStorage/BaseObjectStorageHelper.cs

```diff
@@ -12,24 +12,24 @@
 namespace Microsoft.Toolkit.Uwp.Helpers
 {
     /// <summary>
-    /// Shared implementation of ObjectStorageHelper
+    /// Shared implementation of ObjectStorageHelper.
     /// </summary>
     public abstract class BaseObjectStorageHelper : IObjectStorageHelper
     {
         private JsonSerializer serializer = new JsonSerializer();
 
         /// <summary>
-        /// Gets or sets settings container
+        /// Gets or sets the settings container.
         /// </summary>
         protected ApplicationDataContainer Settings { get; set; }
 
         /// <summary>
-        /// Gets or sets storage folder
+        /// Gets or sets the storage folder.
         /// </summary>
         protected StorageFolder Folder { get; set; }
 
         /// <summary>
-        /// Detect if a setting already exists
+        /// Determines whether if a setting already exists
```

Comment:
Remove the _if_.

Reply:
oops! fixed in 9e7014bdd317d9fc57a9b676ead619c1c9f86e75. I also updated some of the `<returns>` to match the style of ab0a3046d8f045416ff3e8815a5f19e82de0be3c